cmd/cadvisor.go

```diff
@@ -140,9 +142,39 @@ func (ml *metricSetValue) Set(value string) error {
 	return nil
 }
 
+type BlackList map[string]struct{}
+
+func (ms *BlackList) String() string {
```

Comment:
Where are functions with BlackList receiver used?

Reply:
It seems I havn't used String function,so I deleted it.